doc/index.rst

```diff
@@ -1,12 +1,19 @@
+.. Orange3-Associate documentation master file, created by
+   sphinx-quickstart on Fri Jun  3 12:00:36 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
 
-Orange-Associate scripting documentation
-========================================
+Welcome to Orange3-Associate documentation!
+===========================================
+
+Widgets
+-------
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-.. automodule:: fpgrowth
-   :members:
+   widgets/associationrules.rst
+   widgets/frequentitemsets.rst
```

Comment:
Yes, I was confused about that one. All other add-on have two separate folders (scripting and widgets respectively). So everything scripting is in a separate folder in the form of .rst files. So perhaps if I do:

```
Widgets
-----------

 .. toctree::
   :maxdepth: 1

widgets/files.rst

Scripting
------------

 .. toctree::
   :maxdepth: 1

.. automodule:: fpgrowth
   :members:

```

Would it work?